src/formatDate.js

```diff
@@ -7,8 +7,54 @@
  *
  * @returns {string}
  */
+
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dayArr = date.split(fromFormat[3]);
```

Comment:
The variable `dayArr` is misleadingly named since it will contain all parts of the date, not just the day. Consider renaming it to `dateParts` or similar to accurately reflect its contents.